src/components/Round/Round.tsx

```diff
@@ -45,12 +56,18 @@ export const Round: FC<Props> = ({ lang, choices, onAnswer }) => {
   useEffect(() => {
     if (!answer) return;
     onAnswer(answer, time);
+    setShowHint(false);
   }, [answer]);
 
   const onTimerChange = (t: number) => {
     setTime(t);
   };
 
+  const handleHint = (s: string) => {
+    let split = s.split('.');

```

Comment:
`const split` instead
```suggestion
    const split = s.split('.');

```